Logger.Tests/EntityTests.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Logger.Tests;
+
+[TestClass]
+public class EntityTests
+{
+    [TestMethod]
+    public void Students_With_Same_Properties_Are_Equal()
```

Comment:
```suggestion
    public void StudentsRecord_CompareTwoIdenticalRecords_AreEqual()
```
Try to name your tests like this:
`MethodBeingTested_TestingScenerio_ExpectedBehavior`